helix-term/src/ui/editor.rs

```diff
@@ -810,6 +814,29 @@ impl Component for EditorView {
                 status_msg,
                 style,
             );
+        } else {
+            let mut disp = String::new();
+            if let Some(count) = cx.editor.count {
+                disp.push_str(&count.get().to_string())
+            }
+            if let Some(keys) = self.keymaps.pending() {
+                for key in keys {
+                    let s = key.to_string();
+                    if s.width() > 1 {
```

Comment:
I don't think this is a good idea, since some characters are double width like chinese characters.

Reply:
Good catch, but there's this which could mess things up:

https://github.com/helix-editor/helix/blob/88d6f652390922b389667f469b6d308db569bdaf/helix-view/src/input.rs#L45-L46

I think taking grapheme count instead will work.